View.js

```diff
@@ -0,0 +1,131 @@
+function displayExp(skillNr) {
+    /* Sets display of a skill with certain number.
+     *    */
+    getExp(allSkills[skillNr], function (exp) {
+         $(".exp" + skillNr).html(getLevel(exp));
+        });
+}
+
+function createRowTable(skill) {
+    /* Receiving parameters of a skill, creates table row.
+     *    */
+    var htmlCode = (`
+          <h4 class="skill_name"> ` + skill.skillName + `: </h4>
+            <a class="exp` + skill.skillNr + `"> ` + skill.expValue + ` </a>
+            <div>
+              <input id="add_value_num` + skill.skillNr + `" class="add_value_nums" type="number" name="addValue" value="">
+              <button id="add_value_button` + skill.skillNr + `" class="add_value_buttons" type="button">Add</button>
+              <button id="remove_skill_button` + skill.skillNr + `" class="remove_skill_buttons" type="button">Remove</button>
+            </div>
+          `);
+    $('#skills').append(htmlCode);
+    // extension_log(htmlCode);
+}
+
+function displayTable (callback) {
+    /* Creates table of skills by adding every skill row by row
+     *    * and sets display of this skill.
+     *       */
+    for(var i = 0; i < allSkills.length; i++){
+          var skill = {
+                  skillName: allSkills[i],
+                  expValue: -1, // This value will be updated in the display function.
+                  skillNr: i,
+                };
+          createRowTable(skill);
+          displayExp(i);
+        }
+    callback();
+}
+
+function newSkillToTable (nr) {
+    /* Inserts new skill to HTML table of skills.
+     *    */
+    var skill = {
+          skillName: allSkills[nr],
+          expValue: 0,
+          skillNr: nr,
+        };
+    displayExp(nr);
+    createRowTable(skill);
+    handleSkillButtons();
+}
+
+function resetHTMLTable() {
+    $("#skills").remove();
+    $("#skillBody").append(`
+          <div id="skills">
+            </div>
+          `);
+    displayTable(handleSkillButtons);
+}
+
+function handleSkillButtons () {
```

Comment:
Ta funkcja może być wykonana tylko raz, jeżeli w inny sposób będziesz rejestrował eventy do przycisków. Zastosuj Event Delegation.
https://youtu.be/Ef3JYxdFVw4 - tu się dowiesz co to jest
https://learn.jquery.com/events/event-delegation/ - tu jak tego używać w jQuery.

Jeżeli ta poprawka zajmie Ci więcej niż godzinę to ją porzuć.